packages/mailer/src/MailerConfig.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Tempest\Mail;
+
+use Symfony\Component\Mailer\Transport\TransportInterface;
+
+interface MailerConfig
+{
+    /**
+     * The transport class.
+     */
+    public string $transport {
+        get;
+    }
+
+    /**
+     * The default address from which emails will be sent.
+     */
+    public null|string|Address $from {
```

Comment:
Maybe rename to `defaultFrom`?

Reply:
Renamed to `defaultExpeditor` to be even clearer